receiver/prometheusreceiver/internal/otlp_metricfamily.go

```diff
@@ -170,10 +178,23 @@ func (mg *metricGroup) toDistributionPoint(orderedLabelKeys []string, dest *pmet
 	}
 	point.SetTimestamp(tsNanos)
 	populateAttributes(orderedLabelKeys, mg.ls, point.Attributes())
-
+	mg.setExemplars(exemplarsdata, point.Exemplars())
 	return true
 }
 
+func (mg *metricGroup) setExemplars(exemplarMap map[string][]exemplarData, exemplars pmetric.ExemplarSlice) {
+	metricKey := buildMetricKey(mg.ls)
+	ed := exemplarMap[metricKey]
+	exemplars.EnsureCapacity(len(ed))
+	for _, e := range ed {
+		exemplar := exemplars.AppendEmpty()
+		exemplar.SetTimestamp(pcommon.NewTimestampFromTime(time.UnixMilli(e.ts)))
+		exemplar.SetDoubleVal(e.value)
+		for _, l := range e.labels {
+			exemplar.FilteredAttributes().Insert(l.Name, pcommon.NewValueString(l.Value))
```

Comment:
We need to special-case `trace_id` and `span_id`, and add them using SetTraceID() and SetSpanID() per https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#exemplars-1